.eslintrc.js

```diff
@@ -2,20 +2,56 @@ module.exports = {
     root: true,
     parser: '@typescript-eslint/parser',
     plugins: [
-        '@typescript-eslint',
-        'mocha'
+        '@typescript-eslint'
     ],
     extends: [
-        'eslint:recommended',
-        'plugin:@typescript-eslint/recommended',
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
```

Comment:
```suggestion
        'eslint:recommended',
        'plugin:@typescript-eslint/recommended',
```

So that the diff is a touch neater